synth.py

```diff
@@ -179,6 +179,33 @@ def lint_setup_py(session):
     """"mock", "pytest", "pytest-asyncio", "google-cloud-testutils",""",
 )
 
+# Work around https://github.com/googleapis/python-firestore/issues/190
+
+s.replace(
+    "noxfile.py",
+    """\
+    session.run\("coverage", "report", "--show-missing", "--fail-under=100"\)""",
+    """\
+    # Work around https://github.com/googleapis/python-firestore/issues/190
+    session.run("coverage", "report", "--show-missing", "--fail-under=97")""",
+)
```

Comment:
You can pass coverage to the `py_library` and it will adjust the line in noxfile.py

```py
templated_files = common.py_library(
    samples=False,  # set to True only if there are samples
    microgenerator=True,
    cov_level=9,7
)
```

Reply:
8fda6aa